front/app/modules/commercial/representativeness/admin/containers/Dashboard/ChartCards.tsx

```diff
@@ -13,58 +13,37 @@ import { isNilOrError } from 'utils/helperUtils';
 // typings
 import { IUserCustomFieldData } from 'modules/commercial/user_custom_fields/services/userCustomFields';
 
-// fake data
-import fakeData from './fakeData';
+interface Props {
+  projectFilter?: string;
+}
 
-// TODO remove this later, generalize for all select fields
-const onlyGender = ({ attributes: { code } }: IUserCustomFieldData) =>
-  code === 'gender';
+const isSupported = ({ attributes }: IUserCustomFieldData) =>
+  attributes.input_type === 'select';
```

Comment:
Should we also filter out `attributes.code === 'domicile'`? I think that one doesn't work yet on the backend... something to confirm with Adrien

Reply:
Actually the `attributes.input_type === 'select'` might not be necessary here since we're already only requesting 'select'-type fields in the `useUserCustomFields` hook below